### ISSUE TITLE
Release Date

### ISSUE THREAD

**Issue by prabhalar** [pedroigor/keycloak-authz#32]
Hi Pedro - I did a quick test of the app a couple of months back and it looked great. It will add a lot of value to Keycloak if you can merge it with the latest version of keycloak. When can we expect it? 

Thanks,
Raghu

**Comment by cristhiank** [comment]
+1

**Comment by pedroigor** [comment]
Really nice you are able to build and run the project. Thanks for trying it out !

Regarding the release, we are still discussing some bits. We are going to have a meeting next month and one of the topics is this project and the next steps to get it into Keycloak (which is also a big "?", since we still don't have an agreement on how to distribute).

What you saw, is just an initial version of what we are planning. There are a plenty of other features on the roadmap.

Thanks in advance.

**Comment by prabhalar** [comment]
There shouldn't be any questions about merging this with Keycloak as features like UMA and integration with FIDO UAF/U2F will be some of the determining factors for enterprise customers to adopt a product like Keycloak. There are a couple of products that I know of (OpenAM from Forgerock and oxAuth from Gluu) that already have some sort of UMA implementation and you are still catching up with those products

**Comment by pedroigor** [comment]
Yeah, I agree. However, the questions and missing discussions are just around (like I previously said) distribution and how to properly merge it into KC.

Of course, we would need some window to get all those issues (features, etc) done, but that will be the fun part :)

Thanks.
Pedro Igor

**Comment by tmichaud314** [comment]
Has there been any progress made on this issue @pedroigor?  Thank you.

**Comment by pedroigor** [comment]
https://github.com/pedroigor/keycloak/tree/KEYCLOAK-2753
